### ISSUE TITLE
Raiden on Arbitrum: fix timeouts naming convention

### ISSUE THREAD

**Issue by netcriptus** [raiden-network/light-client#3015]
As pointed in [this](https://github.com/raiden-network/raiden-contracts/pull/1573#discussion_r769433236) and [this](https://github.com/raiden-network/raiden-contracts/pull/1573#discussion_r769438688) comments, there's some confusion regarding the naming of timeouts in the Smart Contracts.

There are 2 concepts of time going around, one for absolute timestamp, and another one for number of seconds an event, and the names do not reflect it at all.

Refactor the contracts with better naming, and unifying names that should reflect the same thing.

**Comment by andrevmatos** [comment]
I agree with this, and I think we should take the time to do these refactoring before our next contracts deployment to Arbitrum. But I think this is more of an issue which needs to be tackled first in the contracts, instead of the client, and therefore should be moved there.
